extensions/extensions-helm/aissemble-infrastructure-chart/values.yaml

```diff
@@ -0,0 +1,75 @@
+############################################
+## CONFIG | aiSSEMBLE Infrastructure Configs
+############################################
+
+# Default values for argocd
+argo-cd:
```

Comment:
I: This snuck in due to `.gitignore` changes.  It's basically an inert file as nothing else exists in this directory (not even a POM) so it is harmless.